packages/sdk/src/index.ts

```diff
@@ -3,4 +3,3 @@ export * from './gaming'
 export * from './utils'
 export * from './types'
 export * from './connectors'
-export * from './nft'
```

Comment:
should the gaming one be also removed? (on top)

Reply:
Answered above